generate-protos.sh

```diff
@@ -8,9 +8,11 @@ if [[ "$PROTOC_VERSION" != 'libprotoc 3.6.0' && "$PROTOC_VERSION" != 'libprotoc
 	exit 1
 fi
 PROTOS="mlflow/protos"
+JAVA_OUT="mlflow/java/client/src/main/java"
+mkdir -p "$JAVA_OUT"
```

Comment:
Added `mkdir -p "$JAVA_OUT"` to allow running this script even if `mlflow/java/client/src/main/java` doesn't exist.